### PR TITLE
Feature/alpayk nuc556 transfer checkbox component

### PR DESCRIPTION
**Pull request by alpaykuru**
localhost_8080_components_checkbox html (1)

### CODE REVIEW

**Comment by ahmetceylan** [comment]
there is a typo in props section, it should be "disabled" not "disable"

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "@babel/core": "7.4.0",
     "@babel/preset-env": "7.4.2",
     "@vue/cli-plugin-babel": "3.5.3",
-    "@vue/cli-plugin-e2e-nightwatch": "3.5.1",
+    "@vue/cli-plugin-e2e-nightwatch": "^3.7.0",
```
Comment (osmanertem) [inline package.json]: we should not change this packages version

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (osmanertem) [inline package-lock.json]: we should not need to push package-lock

File: src/components/Checkbox.vue [hunk 3]
```diff
@@ -0,0 +1,185 @@
+<template>
```
Comment (osmanertem) [inline src/components/Checkbox.vue]: there should be a `VxpCheckbox.vue` which uses/extends Checkbox component

**Pull request closed without merging**